apps/dataset-browser/playwright.config.ts

```diff
@@ -0,0 +1,51 @@
+import {defineConfig, devices} from '@playwright/test';
+import {env} from 'node:process';
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig({
+  testDir: './e2e',
+  /* Run tests in files in parallel */
```

Comment:
Minor: use `//` for single-line comments instead (to be in sync with other parts of the codebase)?